tests/test_qiskit_interface.py

```diff
@@ -159,7 +160,7 @@ def test_LiH_dumb_projected_estimator() -> None:
     estimator = Estimator()
     mapper = ParityMapper(num_particles=(1, 1))
 
-    QI = QuantumInterface(estimator, "ErikSD", mapper)
+    QI = QuantumInterface(estimator, "tUCCSD", mapper)
```

Comment:
I have used ErikSD here to speed up the tests. But up to you.

Reply:
I just saw that you removed this Ansatz altogether. RIP :D

Reply:
Ye, I rather just get rid of "dead" code.
I think we have moved on from it xd